### PR TITLE
Feature/proprietor account

### LINKED ISSUE #5: As a proprietor, I want to be able to create a proprietor account (epic-account)

### PR DESCRIPTION
**Pull request by btruong9**
#5 Current version of signup and login pages. Testing for signup/login validation, views, and forms in tests.py. Renders of pages are attached below. 

:sign_up:
:sign_up:proprietor
:login:
:login:proprietor:
:account:
:logout:

### CODE REVIEW

**Review by sharletclaros — DISMISSED** [review]
Nice job on the spacing, based on your demo I know everything is functional. Formatting can be handled after everyone merges. I hope this works. LGTM.

File: sadcowpng/sharedspaces/static/signup.css
Comment (bishalr1) [inline sadcowpng/sharedspaces/static/signup.css:42]: newline needs to be added

File: sadcowpng/sharedspaces/templates/sharedspaces/logout.html
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/logout.html:14]: newline missing

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html:33]: newline missing

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html:30]: Might want to indent the stuff inside form

File: sadcowpng/sharedspaces/urls.py
Comment (bishalr1) [inline sadcowpng/sharedspaces/urls.py:18]: Another newline

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_signup.html
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_signup.html:27]: Pesky newline again

**Review by bishalr1 — DISMISSED** [review]
I found a few nit-picky things but other than that I did not see any obvious bug or required changes. I think it's good to merge once all the conflicts have been resolved.

**Review by sharletclaros — APPROVED** [review]
Spaces removed. Functionality verified. Only merging left.

**Review by bishalr1 — APPROVED** [review]
The merge conflict was resolved and it looks good. Nothing was cut out in the process. LGTM

**Pull request merged**
